manifests/haproxy.pp

```diff
@@ -958,6 +968,23 @@
     }
   }
 
+  if $etcd {
+    haproxy::listen { 'etcd':
+      bind             => $etcd_bind_opts,
+      options          => {
+        'balance' => 'source',
+      },
+      collect_exported => false,
+    }
+    haproxy::balancermember { 'etcd':
+      listening_service => 'etcd',
+      ports             => '4001',
```

Comment:
The official etcd port is 2379 I think, is 4001 because of a clash or something else?